tests/scripts/all.sh

```diff
@@ -629,6 +629,16 @@ component_test_everest () {
     make test
 }
 
+component_test_psa_collect_statuses () {
+  msg "build+test: psa_collect_statuses" # ~30s
+  scripts/config.pl full
+  scripts/config.pl unset MBEDTLS_MEMORY_BUFFER_ALLOC_C # slow and irrelevant
```

Comment:
We may not need to add `unset MBEDTLS_MEMORY_BUFFER_ALLOC_C` if we sideport the removal of MBEDTLS_MEMORY_BUFFER_ALLOC_C from full config. Necessary for now.

Reply:
We definitely should sideport that change. I'd expect it to come around the next time we sync from mbedtls.